src/Microsoft.Azure.SignalR.Common/Auth/MicrosoftEntra/MicrosoftEntraAccessKey.cs

```diff
@@ -195,8 +196,9 @@ private static async Task ThrowExceptionOnResponseFailureAsync(HttpResponseMessa
         var requestUri = response.RequestMessage?.RequestUri?.ToString();
         throw response.StatusCode switch
         {
-            HttpStatusCode.BadRequest => new AzureSignalRInvalidArgumentException(requestUri, innerException, detail),
+            HttpStatusCode.BadRequest => new AzureSignalRInvalidArgumentException(requestUri, innerException, content),
             HttpStatusCode.Unauthorized => new AzureSignalRUnauthorizedException(requestUri, innerException),
```

Comment:
add more into this UnauthorizedException is enough, for example, http status code into it?

Reply:
I am not sure what is the value added here to add kinds of exceptions? Do we expect the customer to catch these exceptions? Or it is just for printing different error messages? If the latter, I feel reuse AzureSignalRException is enough